app/server.go

```diff
@@ -2060,12 +2060,12 @@ func withMut(mut *sync.Mutex, f func()) {
 	f()
 }
 
-func cancelTask(mut *sync.Mutex, task *model.ScheduledTask) {
+func cancelTask(mut *sync.Mutex, task **model.ScheduledTask) {
```

Comment:
This may be confusing. Maybe rename it to taskPointer?

Reply:
Done